docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,274 @@
+---
+title: NGINX Ingress Controller and Istio Service Mesh
+description: |
+  Use NGINX Ingress Controller with Istio Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-889"
+---
+
+## This document is compatible with NGINX Ingress Controller 1.11 and later.    
+
+ NGINX Ingress Controller 1.11 release supports the ability to configure NGINX Ingress CRDs (virtualServer/virtualServerRoute)to use the `service/cluster IP`. To enable NGINX Ingress to route to the `service IP`, we are going to use [use-cluster-ip](https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#upstream).   

```

Comment:
Add the following: 
NGINX Ingress Controller can now be used as the ingress controller for applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX IC provides on Istio-based environments without resorting to any workarounds. 
This is accomplished using the special setting 'use-cluster-ip' for the backend.

Remove line 13 - 15.

Reply:
done